.github/workflows/tests.yml

```diff
@@ -4,20 +4,91 @@ on:
   push:
 
 jobs:
-  build:
+  build-linux:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        platform: [linux]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          wget https://github.com/premake/premake-core/releases/download/v5.0.0-alpha16/premake-5.0.0-alpha16-linux.tar.gz
+          tar -xvf premake-5.0.0-alpha16-linux.tar.gz
+          sudo chmod a+x premake5
+          sudo mv premake5 /usr/local/bin
+
+      - name: Build
+        run: |
+          ./build.sh
+          ./build.sh release
+
+      - name: Tests
+        run: |
+          cd dev
+          ./test.sh
+
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Using wget from powershell take no time (using choco to install it takes 1-2 minutes)
```

Comment:
Extremely annoying that wget isn't installed by default on windows, but only if you use powershell, and even then you'll get a windows version with different arguments...